betamax/adapter.py

```diff
@@ -15,6 +21,7 @@ class BetamaxAdapter(BaseAdapter):
     """
 
     def __init__(self, **kwargs):
+        """__init__ for BetamaxAdapater."""
```

Comment:
I think pep257 0.7.0 separated the error codes for magic methods and non-magic methods so we can ignore this.